### DISCUSSION TITLE
Lang='en' attribute on number field

### DISCUSSION THREAD

**Discussion by WHITE-developer** [solspace/craft-freeform#242] [Questions & Answers] [answered]
Hi,

Is there a reason that the Number field has a predefined `lang='en'` attribute?

For accessibility reasons we would like to remove this attribute on our dutch website, but it looks like this is the only attribute for this field that can't be set dynamically.

https://github.com/solspace/craft3-freeform/blob/master/packages/plugin/src/Fields/NumberField.php#L144

Kind regards,

Rick
WHITE Digital Agency

**Comment by kjmartens** [comment, marked as answer]
Sorry for the trouble Rick. We'll take care of this shortly. :)

**Reply by kjmartens** [reply]
This has been corrected in Freeform 3.12.5

**Reply by WHITE-developer** [reply]
Thanks
